packages/dht/src/connection/websocket/WebsocketConnector.ts

```diff
@@ -193,10 +195,11 @@ export class WebsocketConnector {
                             port: this.selectedPort!,
                             host: this.host,
                             tls: this.config.serverEnableTls,
-                            selfSigned
+                            selfSigned,
+                            version
```

Comment:
There is no `version` field in the protobuf declaration for `ConnectivityRequest`. So remove `version` field here or add it to the protobuf?